components/alert/alert.css

```diff
@@ -0,0 +1,54 @@
+@import url("/index.css");
+@import url("sidebar.css");
+
+*{
+    margin:0;
+    padding: 0;
```

Comment:
0 bananas ??? or 0 lightyear ???

mention units.

Reply:
Should I use px or rem unit?

Reply:
I have made the changes using px. Can you give your feedback . Is it right or not to use px?